test/private/initializeGlobalState.m

```diff
@@ -0,0 +1,47 @@
+function orig = initializeGlobalState()
+% Initialize global state. Set various properties of the graphical root to ensure reliable output of the ACID testsuite.
+% See #542 and #552
+    fprintf('Initialize global state...\n');
+
+    %--- Define desired global state properties
+    % See http://undocumentedmatlab.com/blog/getundoc-get-undocumented-object-properties
+    new.defaultAxesColorOrder.val   =               ...
```

Comment:
I think we have to make it more general: everything in `get(0,'Default')` should match `get(0,'Factory')` during testing (or, I think, equivalently that `get(0,'Default')` should be empty). That doesn't prevent devs from having their own preferences, e.g. as we can reset and restore those values using a similar mechanism as for `get(0)` (which does NOT include those default values!).